content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx

```diff
@@ -0,0 +1,382 @@
+---
+title: 'Baseline your database'
+metaTitle: 'Baseline your database'
+metaDescription: 'Baseline your database using Prisma Migrate in your project'
+langSwitcher: ['typescript', 'node']
+dbSwitcher: ['postgres', 'mysql', 'sqlserver', 'cockroachdb']
+toc: false
+---
+
+## Create an initial migration
+
+To continue evolving your database schema using Prisma Migrate, you will need to [baseline your database](/guides/database/developing-with-prisma-migrate/add-prisma-migrate-to-a-project).
```

Comment:
I think of Prisma Migrate as a "suite of tools" that help with evolving database schemas. Therefore Prisma Migrate  contains `prisma migrate` commands, `db push`, and `db pull`.  

I think this could be a conversation we should have with Product for clarification. We can also have a discussion about this in the DevAdvocacy <> Docs sync too.

Reply:
I think that might be complicated — with the tech switcher 😅